pkg/deploy/api/test/ok.go

```diff
@@ -160,6 +160,36 @@ func OkPodTemplate() *kapi.PodTemplateSpec {
 	}
 }
 
+func OkPodTemplateMissingImageWithOkImageChangeTrigger() *kapi.PodTemplateSpec {
```

Comment:
part of next push (though the above algorithm was not quite right wrt modifying slices within a loop)